### PR TITLE
Update the JSON Schema documentation page #1382

### PR DESCRIPTION
**Pull request by Chinmay922**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

**Comment by Chinmay922** [comment]
I have fixed the following,
 1) Fixed headings which are currently broken
 2) Updated the state to reflect improvements in the project and remaining issues

File: docs/features/jsonSchema.md [hunk 1]
```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
...
 
 * The Old Schema was generated using Jelly files that did not generate a valid JSON Schema.
 * Without a valid schema it is not possible to validate a YAML file rendering the schema unusable.
 * The previous schema included an `"$ref":` for an `object` type, the new draft of the schema makes it mandatory
   to use `"$id":` hence the new schema uses it instead of ref.
-
-== What the schema does
+* The new improvement on [DEV-Tools](https://www.jenkins.io/projects/jcasc/dev-tools/) will be used as an experiment with CommunityBridge and as a project to improve development tools for Jenkins configuration as a code.
+
+ *Issues*
```
Comment (timja) [inline docs/features/jsonSchema.md:49]: This all renders a bit weird, https://github.com/Chinmay922/configuration-as-code-plugin/blob/master/docs/features/jsonSchema.md

I don't think italics are needed, either sub headings or bold would be best imo

File: docs/features/jsonSchema.md [hunk 2]
```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
...
 * The previous schema included an `"$ref":` for an `object` type, the new draft of the schema makes it mandatory
   to use `"$id":` hence the new schema uses it instead of ref.
-
-== What the schema does
+* The new improvement on [DEV-Tools](https://www.jenkins.io/projects/jcasc/dev-tools/) will be used as an experiment with CommunityBridge and as a project to improve development tools for Jenkins configuration as a code.
+
+ *Issues* 
+ * Missing Schema Definitions
+ * Schema validation issues
+ * No automated shcema validation tests
```
Comment (timja) [inline docs/features/jsonSchema.md:52]: there's a testing framework for this?

File: docs/features/jsonSchema.md [hunk 3]
```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
...
-== Improvements
+***Improvements***
 
 * The Old Schema was generated using Jelly files that did not generate a valid JSON Schema.
 * Without a valid schema it is not possible to validate a YAML file rendering the schema unusable.
 * The previous schema included an `"$ref":` for an `object` type, the new draft of the schema makes it mandatory
   to use `"$id":` hence the new schema uses it instead of ref.
-
-== What the schema does
+* The new improvement on [DEV-Tools](https://www.jenkins.io/projects/jcasc/dev-tools/) will be used as an experiment with CommunityBridge and as a project to improve development tools for Jenkins configuration as a code.
```
Comment (timja) [inline docs/features/jsonSchema.md:47]: not sure if this is needed, the tense is wrong anyway this was completed a long time ago

File: docs/features/jsonSchema.md [hunk 4]
```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
 The new JSON Schema is structured to validate the YAML files that are loaded via JCasC.
 The structure and validation of the Schema is done based on the user-installed plugins.
 
-=== Version
+ ***Version***
```
Comment (jetersen) [inline docs/features/jsonSchema.md:22]: I do not agree with italics and bold styling.

Use headers:
```md
# Using JSON Schema

## Using JSON Schema

### Using JSON Schema

#### Using JSON Schema

##### Using JSON Schema

###### Using JSON Schema
```

# Using JSON Schema

## Using JSON Schema

### Using JSON Schema

#### Using JSON Schema

##### Using JSON Schema

###### Using JSON Schema

File: docs/features/jsonSchema.md [hunk 5]
```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
...
 
 The Schema uses JSON draft v07. 
 http://json-schema.org/draft-07/schema#
 
-=== How to use
+***Using JSON Schema***
```
Comment (jetersen) [inline docs/features/jsonSchema.md:27]: Ditto for all the `***` changes

Comment (Chinmay922) [reply docs/features/jsonSchema.md]: I changed all the sub-heading with italic texts to simple bold texts.
Could you please check and let me know?

**Pull request closed without merging**